src/igvfd/types/file.py

```diff
@@ -112,6 +112,23 @@ def href(self, request, file_format, accession):
             filename
         )
 
+    @calculated_property(
+        schema={
+            'title': 'S3 URI',
+            'description': 'The S3 URI of public file object.',
```

Comment:
These files aren't public. I'm not sure in general if it's a great idea to expose our internal bucket names, when they can't be accessed publicly.

It's important to remember that we only added s3_uri field to ENCODE after we had switched to the public dataset. Somehow checkfiles worked without it.

On the plus side at least this is still a form of relying on the portal to tell checkfiles where to find a file (e.g. will be simpler if files in another S3 bucket or even a GC bucket/ANViL project ever need to be checked).

Unfortunate though that the streaming checkfiles didn't work by relying directly on the portal access abstraction, instead of having information about the underlying buckets. Now code and credentials will have to be duplicated/maintained across portal/checkfiles if there are ever additional buckets/cloud providers added.

Reply:
Yea. Worrying about other cloud providers etc should be worried when they get added though.

Reply:
You should update the description though.